### PR TITLE
Routes redesign

### PR DESCRIPTION
**Pull request by atstoyanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/models/application.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum ApplicationTypeEnum {
```
Comment (hero101) [inline src/models/application.ts:1]: the file name must be at least 'application-type'.

File: src/pages/Admin/Community/CommunityListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { Loading } from '../../../components/core';
+import { useDeleteUserGroup } from '../../../hooks';
+import { useCommunityGroupsQuery } from '../../../hooks/generated/graphql';
+import { PageProps } from '../../common';
+
+interface CommunityListPageProps extends PageProps {
+  communityId: string;
+}
+
+export const CommunityListPage: FC<CommunityListPageProps> = ({ paths, communityId }) => {
```
Comment (hero101) [inline src/pages/Admin/Community/CommunityListPage.tsx:13]: This is Community group list, not community list

File: src/pages/Admin/Community/CommunityListPage.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const { url } = useRouteMatch();
+  const { data, loading } = useCommunityGroupsQuery({
```
Comment (hero101) [inline src/pages/Admin/Community/CommunityListPage.tsx:15]: community query will be removed in the future

**Review by hero101 — CHANGES_REQUESTED** [review]

File: graphql/queries/communityGroups.graphql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+query communityGroups($communityId: UUID!) {
+  community(ID: $communityId) {
```
[the remark this answers is not in the document]
Comment (techsmyth) [reply graphql/queries/communityGroups.graphql:2]: If possible indeed use another query so we do not build up debt.

File: src/hooks/user/useUserMetadataWrapper.ts [hunk 5]
```diff
@@ -60,13 +61,26 @@ export const useUserMetadataWrapper = () => {
...
+      const isEcoverseAdmin = (id: string) =>
+        hasCredentials(AuthorizationCredential.GlobalAdmin) ||
+        hasCredentials(AuthorizationCredential.EcoverseAdmin, id);
+
+      const isChallengeAdmin = (ecoverseId: string, challengeId: string) =>
+        isEcoverseAdmin(ecoverseId) || hasCredentials(AuthorizationCredential.ChallengeAdmin, challengeId);
+
+      const isOpportunityAdmin = (ecoverseId: string, challengeId: string, opportunityId) =>
+        isChallengeAdmin(ecoverseId, challengeId) ||
+        hasCredentials(AuthorizationCredential.OpportunityAdmin, opportunityId);
```
[the remark this answers is not in the document]
Comment (atstoyanov) [reply src/hooks/user/useUserMetadataWrapper.ts:74]: If you are admin of an opportunity's parent challenge, but you are not member of the opportunity, are you admin of that opportunity?

File: src/pages/Admin/Community/CommunityListPage.tsx (hunk 3 above)
Comment (atstoyanov) [reply src/pages/Admin/Community/CommunityListPage.tsx:17]: Replaced with the new ecoverse.community query

File: src/hooks/useAvailableMembers.ts [hunk 6]
```diff
@@ -48,12 +49,26 @@ export const useAvailableMembers = (
       },
     },
   });
+
+  const {
+    data: _parentCommunityMembers,
+    loading: loadingParentCommunityMembers,
+    error: parentCommunityMembersError,
+  } = useCommunityMembersQuery({
```
[the remark this answers is not in the document]
Comment (atstoyanov) [reply src/hooks/useAvailableMembers.ts:59]: Updated!

File: src/hooks/user/useUserMetadataWrapper.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (techsmyth) [reply src/hooks/user/useUserMetadataWrapper.ts:74]: Yes you are, the authorizations cascade down.

**Review by hero101 — CHANGES_REQUESTED** [review]
Two convos are not addressed.

**Comment by hero101** [comment]
> Two convos are not addressed.

How is [this discussion](https://github.com/alkem-io/client-web/pull/1206#discussion_r720240136) going to be addressed?

**Review by hero101 — APPROVED** [review]

**Pull request merged**